lib/spack/spack/util/file_cache.py

```diff
@@ -42,7 +42,7 @@ class FileCache(object):
 
     """
 
-    def __init__(self, root):
+    def __init__(self, root, default_timeout=120):
```

Comment:
rename to just `timeout` -- if we're setting it through a parameter it's not the default timeout.